config/initializers/active_admin.rb

```diff
@@ -3,9 +3,11 @@
   #
   # Set the title that is displayed on the main layout
   # for each of the active admin pages.
-  #
   config.site_title = "Baja Pet Rescue"
 
+  # Load the Google API JS library
+  config.register_javascript 'https://www.google.com/jsapi'
```

Comment:
Could you tell me a bit more how this gets used?

Reply:
This is for registering Google Chart JavaScript to active_admin.  Chartkick gem is a wrapper for several of the most popular open source chart libraries like Google Chart and Chart.js. Without this line, it does not work.

Reply:
Oh I see, ok that makes sense. Thank you.